contrib/cirrus/setup.sh

```diff
@@ -26,4 +26,7 @@ curl --fail --location -o /tmp/netavark.zip "$NETAVARK_URL"
 mkdir -p /usr/libexec/podman
 cd /usr/libexec/podman
 unzip /tmp/netavark.zip
+if [[ $(uname -m) != "x86_64" ]]; then
```

Comment:
May need a `rm -f netavark` before the unzip line.   Or tell unzip to overwrite everything.